float64 xpos
float64 ypos
float64 zpos

int16   sw

float64 xpos_ref
float64 ypos_ref
float64 zpos_ref
